ui/tests/acceptance/secrets/backend/aws/aws-configuration-test.js

```diff
@@ -266,54 +261,18 @@ module('Acceptance | aws | configuration', function (hooks) {
       await runCmd(`delete sys/mounts/${path}`);
     });
 
-    test('it should not make a post request if lease or root data was unchanged', async function (assert) {
```

Comment:
No longer relevant since we opted not to do dirty tracking with the new form classes.

Reply:
this answers my question above!